sdk-api-src/content/uiautomationcoreapi/nf-uiautomationcoreapi-uiaraisenotificationevent.md

```diff
@@ -53,6 +53,8 @@ api_name:
 
 Called by providers to initiate a notification event.
 
+To support Narrator announcing text your root view must be able to create an automation peer or you need to set its landmark type to [UIA_MainLandmarkTypeId](../winauto/landmark-type-identifiers). Also if your window is created using the `WS_POPUP` style you'll have to handle the [WM_GETOBJECT](../winauto/wm-getobject) message ([example](../winauto/uiauto-howto-expose-serverside-uiautomation-provider)) and implement the [Window Control Pattern](../winauto/uiauto-implementingwindow).
```

Comment:
I would remove the first sentence. There is no concept as a view in UIA (only UIA providers which loosely translate to automation peers, and this API already expects you to pass a UIA provider), and the landmark type workaround isn't something that is guaranteed to not change so I would not document this. Instead:

```suggestion
If your window is created using the [`WS_POPUP`](/windows/win32/winmsg/window-styles) style you'll have to handle the [WM_GETOBJECT](../winauto/wm-getobject) message ([example](../winauto/uiauto-howto-expose-serverside-uiautomation-provider)) and implement the [Window Control Pattern](../winauto/uiauto-implementingwindow).
```

Reply:
CC @fabraham-ms who owns these APIs, can you take a look?

Reply:
OK, I updated it. It would be nice to document the landmark type workaround somewhere, but I guess for RNW Islands users at least they have that code in the playground-win32 app to follow for now.